doc/howtos/image-configuration.rst

```diff
@@ -0,0 +1,100 @@
+Configuring Refkit images
```

Comment:
We agreed with Ismo that I can somehow merge this to development story at some point